### ISSUE TITLE
Add more examples

### ISSUE THREAD

**Issue by cmungall** [mapping-commons/sssom#229]
On with Richard on TIMS call, we think it would be useful to have a few more examples, on the site, in the examples/ folder, or both

The existing examples seem quite minimal which is useful for illustrating the basic core concepts, internal vs external mode.

But it would be useful to have more examples that show the union of features all in one, perhaps with a diverse range of mapping predicates

**Comment by gouttegd** [comment]
I don’t think this is relevant anymore. We have many more examples than we used to have.
